client/my-sites/stats/features/modules/types.d.ts

```diff
@@ -59,6 +59,7 @@ type StatsPeriodType = {
 type StatsQueryType = {
 	date: string;
 	period: StatsPeriodGrainType;
+	geoMode?: 'country' | 'region' | 'city';
```

Comment:
Do you think [this approach](https://github.com/Automattic/wp-calypso/pull/103241/files) is a reasonable one for an alternative here?

Reply:
Addressed in 47fd208baf8